test/testdata/autogen/self_class.rb.autogen.exp

```diff
@@ -0,0 +1,51 @@
+# ParsedFile: test/testdata/autogen/self_class.rb
+requires: []
+## defs:
+[def id=0]
+ type=module
+ defines_behavior=0
+ is_empty=0
+[def id=1]
+ type=module
+ defines_behavior=0
+ is_empty=1
+ defining_ref=[DSL]
+[def id=2]
+ type=class
+ defines_behavior=1
+ is_empty=0
+ defining_ref=[A]
+[def id=3]
+ type=class
+ defines_behavior=1
+ is_empty=1
+ defining_ref=[B]
+## refs:
```

Comment:
But the list of refs here does not include that constant (note that none of the refs are online 7)